.github/workflows/CI.yml

```diff
@@ -64,6 +64,7 @@ jobs:
         .\redis-server.exe --service-install --service-name "redis-6379" "..\Basic\primary-6379-3.0.conf"
         .\redis-server.exe --service-install --service-name "redis-6380" "..\Basic\replica-6380.conf"
         .\redis-server.exe --service-install --service-name "redis-6381" "..\Basic\secure-6381.conf"
+        .\redis-server.exe --service-install --service-name "redis-6384" "..\Basic\tls-ciphers-6384.conf"
```

Comment:
This might not be necessary - I pushed it up but I'm assuming this is pointing at redis 3?

Reply:
oh yeah good call - let me roll this back when I get a sec, it won't work so let's remove.